db/db_impl_write.cc

```diff
@@ -1104,41 +1107,72 @@ Status DBImpl::HandleWriteBufferFull(WriteContext* write_context) {
   // no need to refcount because drop is happening in write thread, so can't
   // happen while we're in the write thread
   ColumnFamilyData* cfd_picked = nullptr;
+  std::vector<std::vector<uint32_t>> to_flush;
+  // Elements in cfds must be unique.
+  autovector<ColumnFamilyData*> cfds;
```

Comment:
Not sure if a undordered_set is better here ? What do you think ?

Reply:
Well that depends. We ensure RocksDB runs in an environment with low memory capacity. Sometimes we intentionally trade search performance for memory efficiency. In this case, the number of active column families is not high, thus I think we can do with linear search. Let me know if this is not the case in your situation.